app/dao/notifications_dao.py

```diff
@@ -136,6 +135,28 @@ def dao_get_7_day_agg_notification_statistics_for_service(service_id,
 
 
 @statsd(namespace="dao")
+def dao_get_template_usage(service_id, limit_days=None):
+
+    query = db.session.query(
+        func.count(NotificationHistory.template_id).label('count'),
+        NotificationHistory.template_id,
+        func.DATE(NotificationHistory.created_at).label('day'),
+        Template.name,
+        Template.template_type
+    )
+
+    query_filter = [NotificationHistory.service_id == service_id]
+    if limit_days is not None:
+        query_filter.append(NotificationHistory.created_at >= days_ago(limit_days))
+
+    return query.filter(*query_filter) \
+        .join(Template)\
+        .group_by(NotificationHistory.template_id, func.DATE(NotificationHistory.created_at), Template.name, Template.template_type)\
```

Comment:
do we need to group by day for any reason? Given that we're changing the API schema anyway, I feel like we can take the extra step and just group by template_id - that then gets rid of that silly aggregate_usage function on the admin side